microscopium/preprocess.py

```diff
@@ -677,6 +677,47 @@ def _reservoir_sampled_image(ims_iter, random_state=None):
     return sampled
 
 
+def global_threshold(ims_iter, random_state=None):
+    """Generate a global threshold for the collection of images given.
+
+    The threshold is determined by sampling the intensity of every
+    image and then computing the isodata [1]_ threshold on this sample.
+
```

Comment:
For the sake on consistency, should isodata supplant the otsu and/or adaptive threshold that are used by default in ``intensity_object_features``?